pkg/metrics/constants.go

```diff
@@ -33,6 +33,9 @@ const (
 	// Reasons for CREATE/DELETE shared metrics
 	ProvisionedReason = "provisioned"
 	ExpiredReason     = "expired"
+
+	// Response for eviction request
+	ResponseLabel = "response"
```

Comment:
```suggestion
	CodeLabel = "code"
```

Consider calling this `code` like `operatorpkg` does with its metric labels